server/app/views/applicant/ProgramIndexView.java

```diff
@@ -286,17 +286,21 @@ private LiTag programCard(
               preferredLocale, cardData.latestSubmittedApplicationStatus().get()));
     }
     programData.with(title, description);
-
-    // Add info link.
-    String infoUrl =
-        controllers.applicant.routes.ApplicantProgramsController.view(applicantId, program.id())
-            .url();
+    // Add external link if it is present else use the default Program details page
```

Comment:
**suggestion:** Consider updating to something like: "Navigate to the external link if it is set; otherwise, navigate to the default program details page"

Reply:
Navigation happens after the button click, we are just using this here.